doc/whats_new.rst

```diff
@@ -108,6 +108,10 @@ Bug fixes
      ``pandas.Series`` in their ``fit`` function. :issue:`7825` by
      `Kathleen Chen`_.
 
+   - Fix a bug where :class:`sklearn.ensemble.VotingClassifier` raises an error
+     when numpy array is passed in for weights. :issue:`7983` by
```

Comment:
"a numpy array"? btw, there is now an easier way to add a link to your github account using the ``:user:`` directive